sentinelone/plugin.spec.yaml

```diff
@@ -1050,84 +1001,101 @@ actions:
         title: Site IDS
         description: List of Site IDs to filter by
         type: '[]string'
         required: false
+        example: ["5000000000000001"]
       agent_ids:
         title: Agent IDS
         description: Return activities related to specified agent ids
         type: '[]string'
         required: false
+        example: ["9000000000000000"]
       skip_count:
         title: Skip Count
         description: If true, total number of items will not be calculated, which speeds up execution time
         type: boolean
         required: false
+        example: true
       ids:
         title: Activity IDS
-        description: If true, total number of items will not be calculated, which speeds up execution time
+        description: List of Activity IDs
         type: '[]string'
         required: false
+        example: ["800000000000000008"]
       created_at_lt:
         title: Less Then Date
-        description: Return activities created before this date in ISO-8601, example 2018-02-27T04:49:26.257525Z
+        description: Return activities created before this date in ISO-8601
         type: string
         required: false
+        example: "2020-12-20T18:49:26.257525Z"
       created_at_lte:
         title: Less or Equal Date
-        description: Return activities created before or at this date in ISO-8601, example 2018-02-27T04:49:26.257525Z
+        description: Return activities created before or at this date in ISO-8601, example 2020-12-18T18:49:26.257525Z
         type: string
         required: false
+        example: "2020-12-20T18:49:26.257525Z"
       created_at_gt:
         title: Greater Then Date
-        description: Return activities created after or at this date in ISO-8601, example 2018-02-27T04:49:26.257525Z
+        description: Return activities created after or at this date in ISO-8601, example 2020-12-18T18:49:26.257525Z
         type: string
         required: false
+        example: "2020-12-18T18:49:26.257525Z"
       created_at_gte:
         title: Greater or Equal Date
-        description: Return activities created after or at this date in ISO-8601, example 2018-02-27T04:49:26.257525Z
+        description: Return activities created after or at this date in ISO-8601, example 2020-12-18T18:49:26.257525Z
         type: string
         required: false
+        example: "2020-12-20T18:49:26.257525Z"
       created_at_between:
         title: Between Dates
         description: Return activities created within this range (inclusive), example 1514978764288-1514978999999
         type: string
         required: false
+        example: "1514978764288-1514978999999"
       cursor:
         title: Cursor Position
-        description: Cursor position returned by the last request. Should be used for iterating over more than 1000 items, example YWdlbnRfaWQ6NTgwMjkzODE=
+        description: Cursor position returned by the last request. Should be used for iterating over more than 1000 items
```

Comment:
Can we handle pagination automatically for the user within the code? Having the user handle pagination w/in the workflow builder can be clunky.